omas/omas_utils.py

```diff
@@ -864,23 +865,6 @@ def omas_global_quantities(imas_version=omas_rcparams['default_imas_version']):
     return _global_quantities[imas_version]
 
 
-# only attempt cython if effective user owns this copy of omas
-# disabled for Windows: need to add check for file ownership under Windows
-if os.name == 'nt' or os.geteuid() != os.stat(__file__).st_uid:
```

Comment:
With this change the compiled version of `omas_cython` should be available to everyone. Also windows users.